platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/activities/commands/SendCommand.java

```diff
@@ -90,4 +95,8 @@ private void sendActivity() throws Exception {
             }
         }
     }
+
+    public void setBundleContext(BundleContext bundleContext) {
```

Comment:
:+1: I think that it might only ever be used in unit tests, and this module doesn't have any unit tests.